### ISSUE TITLE
Error link_external_mxf when using opatom video

### ISSUE THREAD

**Issue by haraldjordan78** [markreidvfx/pyaaf2#55]
Hi,

i am a little unexperienced otherwise i would have loved to open a PR. 
Got this error message when using ama_link or link_external_mxf. 
I believe line 721 in mxf.py should look like this:
```
FALSE: n['slot_id'].value = self.data['URLString']
CORRECT: n['URLString'].value = self.data['URLString']
```

My code:

```
with aaf2.open(sys.argv[1]+".aaf", 'w') as f:
    f.content.link_external_mxf("C:\\dev\\python\\opatom\\GUMO_1906V01C478F96E.mxf")
```

Message: 
Traceback (most recent call last):
  File "C:\dev\python\create_linked.py", line 22, in <module>
    f.content.link_external_mxf("C:\\dev\\python\\opatom\\GUMO_1906V01C478F96E.mxf")
  File "C:\Python27\lib\site-packages\aaf2\content.py", line 75, in link_external_mxf
    return m.link(self.root)
  File "C:\Python27\lib\site-packages\aaf2\mxf.py", line 831, in link
    mobs.append(package.link())
  File "C:\Python27\lib\site-packages\aaf2\mxf.py", line 262, in link
    mob.descriptor = d.link()
  File "C:\Python27\lib\site-packages\aaf2\mxf.py", line 587, in link
    d['Locator'].append(item.link())
  File "C:\Python27\lib\site-packages\aaf2\mxf.py", line 721, in link
    n['slot_id'].value = self.data['URLString']
  File "C:\Python27\lib\site-packages\aaf2\core.py", line 330, in __getitem__
    raise KeyError(key)
KeyError: u'slot_id'

**Comment by markreidvfx** [comment]
Thanks for reporting the issue. It should be fixed now in 47026ff40c03a422cf1c6344ec45125b6bf45b10.

**Comment by haraldjordan78** [comment]
Very nice, thanks a lot!
